superdesk/activity.py

```diff
@@ -121,7 +133,8 @@ class ActivityResource(Resource):
     }
     exclude = {endpoint_name, 'notification'}
     datasource = {
-        'default_sort': [('_created', -1)]
+        'default_sort': [('_created', -1)],
+        'filter': {'_created': {'$gte': utcnow() - datetime.timedelta(days=1)}}
```

Comment:
I have been told that only last 24h of activities are relevant so return to the users so I have put this  in. Also Michael applied capped collection script to Activity collection so that script will be shared with team Europe.